src/main/frontend/config.cljs

```diff
@@ -44,7 +44,9 @@
       (def REGION "us-east-1")
       (def USER-POOL-ID "us-east-1_dtagLnju8")
       (def IDENTITY-POOL-ID "us-east-1:d6d3b034-1631-402b-b838-b44513e93ee0")
-      (def OAUTH-DOMAIN "logseq-prod.auth.us-east-1.amazoncognito.com"))
+      (def OAUTH-DOMAIN "logseq-prod.auth.us-east-1.amazoncognito.com")
+      (def CONNECTIVITY-TESTING-S3-URL "https://logseq-connectivity-testing-prod.s3.us-east-1.amazonaws.com/logseq-connectivity-testing")
```

Comment:
I personally prefer to use the same URL base to test connectivity.
Connectivity testing can be disabled in dev mode. (as the dev mode is not publicly available to end users)

CC @cnrpman

Reply:
updated